Sources/TuistCore/ContentHashing/ContentHasher.swift

```diff
@@ -51,12 +57,12 @@ public final class ContentHasher: ContentHashing {
         return try hash(dictString)
     }
 
-    public func hash(path filePath: AbsolutePath) throws -> String {
+    public func hash(path filePath: AbsolutePath) async throws -> String {
         if fileHandler.isFolder(filePath) {
-            return try fileHandler.contentsOfDirectory(filePath)
+            return try await fileHandler.contentsOfDirectory(filePath)
                 .filter { filesFilter($0) }
                 .sorted(by: { $0 < $1 })
-                .map { try hash(path: $0) }
+                .concurrentMap { try await hash(path: $0) }
```

Comment:
This should give us a performance improvement.

Reply:
Yeah, I was also thinking these `concurrentMap`s that are now used when doing file-based operations should give us a boost 🤞